### DISCUSSION TITLE
Best way to simulate flexible surface

### DISCUSSION THREAD

**Discussion by mandralis** [google-deepmind/mujoco#2177] [Asking for Help]
### Intro

Hi!

I am a graduate student at Caltech.

### My setup

MuJoCo 3.2.4 on Ubuntu 20.04

### My question

I am trying to simulate a flexible surface using Mujoco. This surface is a shaped piece of mylar plastic that can bend but is not soft like fabric. Ideally I want to simulate this from a .stl mesh which defines the surface geometry. Can somebody help me achieve this ? 

I have already tried using the flexbody with type=mesh element. However the mujoco simulator simply does not load anything and freezes and crashes. 

Best regards,
Ioannis 

### Minimal model and/or code that explain my question

Model:

```
<mujoco model="flexcomp_mesh_example">
   <compiler angle="degree" coordinate="local"/>
   <option timestep="0.002" gravity="0 0 -9.81"/>

   <asset>
       <!-- Optional material and texture assets if needed -->
       <texture name="tex" type="cube" builtin="checker" width="128" height="128" rgb1="0.2 0.3 0.4" rgb2="0.1 0.2 0.3"/>
       <material name="mat" texture="tex" reflectance="0.5"/>
   </asset>

   <worldbody>
       <!-- Ground plane -->
       <geom type="plane" pos="0 0 0" size="10 10 0.1" rgba="0.8 0.8 0.8 1"/>

       <!-- Flexcomp body generated from the mesh -->
       <flexcomp name="flexcomp_body" type="mesh" file="sheet.stl" pos="0 0 0.5" radius="0.02" 
                 rigid="false" mass="1" scale="1 1 1" rgba="0.5 0.2 0.2 1"/>
   </worldbody>

   <actuator>
       <!-- Define actuators if needed -->
   </actuator>
</mujoco>

```

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by quagla** [comment]
Hi, how big is the mesh? Do you get any log with the crash and have you looked at the memory occupation?
